src/components/progress-flow/progress-flow-item/progress-flow-item.scss

```diff
@@ -133,7 +133,7 @@
     z-index: z-index.$limel-progress-flow-divider;
```

Comment:
I'd like the commit message to be formatted something like `fix(progress-flow): remove scrollbar that sometimes appears', but I'll just merge this into a new branch in this repo, and update it myself before merging to `next` 👍